eng/Versions.props

```diff
@@ -65,6 +65,9 @@
     <MicrosoftDotNetGenAPIPackageVersion>5.0.0-beta.20171.1</MicrosoftDotNetGenAPIPackageVersion>
     <!-- Packages from dotnet/roslyn -->
     <MicrosoftNetCompilersToolsetPackageVersion>3.6.0-3.20170.19</MicrosoftNetCompilersToolsetPackageVersion>
+    <!-- Packages from dotnet/roslyn-sdk -->
+    <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20178.1</MicrosoftCodeAnalysisTestingVersion>
```

Comment:
Where is the Microsoft.CodeAnalysis.Testing package used?

Is it just a transitive reference that Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.XUnit pulls in? If yes, it need not be mentioned here.

Reply:
Microsoft.CodeAnalysis.Testing is a shared version number that feeds in to 6 packages:

1. Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.XUnit
1. Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.XUnit
1. Microsoft.CodeAnalysis.CSharp.CodeRefactoring.Testing.XUnit
1. Microsoft.CodeAnalysis.VisualBasic.Analyzer.Testing.XUnit
1. Microsoft.CodeAnalysis.VisualBasic.CodeFix.Testing.XUnit
1. Microsoft.CodeAnalysis.VisualBasic.CodeRefactoring.Testing.XUnit

All of these should use the same version.